adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5']
+=== Release 1.5, September 2019
+
+[id='sec.1_5.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.18.0.
+
+[id='sec.1_5.issue']
+==== Known Issues
+
+IMPORTANT: In order to deploy to this version, or upgrade from {scf} 2.17.1, part of {cap} 1.4.1, to this version, you will need to scale down the database roles to 1 pod in order to migrate from cf-mysql to PXC. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment
```

Comment:
Upgrades or deploy : you will need to scale down the database roles to 1 pod 
only upgrades: in order to migrate from cf-mysql to PXC. This is based on upstream instructions
Only Deploy: but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment

Reply:
We should have the wording better, noting that both deployment and upgrade need mysql role set to 1.